src/DonationSummary/resources/js/summary.js

```diff
@@ -1,63 +1,93 @@
-        $form.find('.fee-break-down-message').hide()
-        $form.find('.js-give-donation-summary-fees').toggle(targetNode.checked)
+    initFees: function() {
+        GiveDonationSummary.observe('.give_fee_mode_checkbox', function (targetNode, $form) {
+            $form.find('.fee-break-down-message').hide()
+            $form.find('.js-give-donation-summary-fees').toggle(targetNode.checked)
 
-        // Hack: (Currency Switcher) The total is always stored using a the decimal seperator as set by the primary currency.
-        const fee = document.querySelector('[name="give-fee-amount"]').value.replace('.', Give.form.fn.getInfo( 'decimal_separator', $form ))
-        $form.find('[data-tag="fees"]').html(
-            GiveDonationSummary.format_amount(fee, $form)
-        )
-    })
+            // Hack: (Currency Switcher) The total is always stored using a the decimal seperator as set by the primary currency.
+            const fee = document.querySelector('[name="give-fee-amount"]').value.replace('.', Give.form.fn.getInfo( 'decimal_separator', $form ))
+            $form.find('[data-tag="fees"]').html(
+                GiveDonationSummary.format_amount(fee, $form)
+            )
+        })
+    },
 
     /**
-     * Total
      * @unreleased
      */
-    GiveDonationSummary.observe( '.give-final-total-amount', function( targetNode, $form ) {
-        // Hack: (Currency Switcher) The total is always stored using a the decimal seperator as set by the primary currency.
-        const total = targetNode.dataset.total.replace('.', Give.form.fn.getInfo( 'decimal_separator', $form ))
-        $form.find( '[data-tag="total"]' ).html(
-            GiveDonationSummary.format_amount( total, $form )
-        )
-    })
-
-    // Hack: Force an initial mutation for the Total Amount observer
-    const totalAmount = document.querySelector('.give-final-total-amount')
-    if( totalAmount ){
-        totalAmount.dataset.total = totalAmount.dataset.total
-    }
-})
+    initTotal: function() {
+        GiveDonationSummary.observe('.give-final-total-amount', function (targetNode, $form) {
+            // Hack: (Currency Switcher) The total is always stored using a the decimal seperator as set by the primary currency.
+            const total = targetNode.dataset.total.replace('.', Give.form.fn.getInfo('decimal_separator', $form))
+            $form.find('[data-tag="total"]').html(
+                GiveDonationSummary.format_amount(total, $form)
+            )
+        })
+      
+      // Hack: Force an initial mutation for the Total Amount observer
+      const totalAmount = document.querySelector('.give-final-total-amount')
+      if( totalAmount ){
+      totalAmount.dataset.total = totalAmount.dataset.total
+      }
+    },
 
-/**
- * @unreleased
- */
-const GiveDonationSummary = {
+    /**
+     * Hack: Placeholder callback, which is only used when the gateway changes.
+     */
+    handleNavigateBack: function() {},
```

Comment:
This should probably be removed.

Reply:
@alaca this is actually here intentionally. Perhaps the comment isn't clear (even I forgot when reviewing it just now). The `handleNavigateBack` only needs to be set after the gateway has changed. Prior to that the Sequoia template handles it correctly.

Reply:
Right 😅